packages/storybook/src/stories/grid-sorting.stories.tsx

```diff
@@ -432,3 +435,58 @@ export const ServerSideSorting = () => {
     </div>
   );
 };
+
+export const ResetSortingRows = () => {
+  const columns = [
+    {
+      field: 'name',
+      width: 200,
+    },
+    {
+      field: 'team',
+      width: 200,
+    },
+  ];
+  const [rows, setRows] = React.useState<RowsProp>([]);
+
+  const createRandomRows = () => {
+    const randomRows: any[] = [];
+
+    for (let i = 0; i < 10; i += 1) {
+      const id = randomInt(0, 100000).toString();
+      randomRows.push({ id, name: 'name test', team: id });
+    }
+
+    setRows(randomRows);
+  };
+
+  return (
+    <div
+      style={{
+        height: '1000px',
+      }}
+    >
+      <Button onClick={() => createRandomRows()}>Random Rows</Button>
+      <XGrid
+        rows={rows}
+        columns={columns}
+        loading={rows.length === 0}
+        rowHeight={56}
+        hideFooter
+        showCellRightBorder
+        showColumnRightBorder
+        disableExtendRowFullWidth
+        onColumnHeaderClick={() => {}}
+        sortModel={[
+          {
+            field: 'name',
+            sort: 'asc',
+          },
+        ]}
+        components={{
+          noRowsOverlay: () => <GridOverlay>No Data</GridOverlay>,
+        }}
```

Comment:
What is it for?
```suggestion
```

Reply:
yeah useless, I just dropped the codesandbox